modules/hub/src/main/Types.scala

```diff
@@ -0,0 +1,8 @@
+package lila.hub
+
+package object tournamentTeam {
+  type TeamId = String;
+  type TeamName = String;
+  type TeamIdList = List[TeamId];
+  type TeamIdsWithNames = List[(TeamId, TeamName)]
+}
```

Comment:
type aliases don't improve type safety, only code readability.

I can pass a FEN string to a function that expects a `TeamId` and it will accept it, because it's just a String.

What improves type safety are wrappers like `case class TeamId(value: String) extends AnyVal`. Now you can't pass a name where an Id is expected.

It does however add boilerplate and I don't think it's necessary here, so we'll keep it like that.